### PR TITLE
[CPS-2024] - Update sponsors, speakers, and event schedule

### PR DESCRIPTION
**Pull request by theer1k**
This PR adds the following details for the upcoming DevOpsDays Campinas event:

## Organizers:
 - Erik Henrique
 
## Sponsors:
- AWS
- UNISAL
- BeTech
- Novatec
- 3DComTech
- AWS User Group Campinas

## Event Schedule:
- **09:00 - 09:40:** Credenciamento e Coffee
- **09:50 - 10:00:** Abertura
- **10:00 - 10:50:** Patricia Tozi - *DataOps: A Nova Fronteira para o Big Data e IA no Mundo DevOps*
- **10:50 - 11:30:** Renicius Pagotto Fostaini
- **11:30 - 12:10:** Bianca Veit da Silva - *Testes de Chaos no iFood: Explorando Litmus Chaos e Kubernetes*
- **12:10 - 13:40:** Almoço
- **13:10 - 13:40:** Ignite Karaoke
- **13:40 - 14:20:** Julia Pedrosa - *Platform Engineering para Iniciantes*
- **14:20 - 15:00:** Lucas Ribeiro Mata - *MLOps: Automatizando e Otimizando Projetos de Visão Computacional*
- **15:00 - 15:40:** Levi Leopoldino - *Construindo uma Carreira em DevOps: Tendências e Oportunidades no Mercado*
- **15:40 - 16:10:** Coffee Break
- **16:10 - 16:50:** Open Space
- **16:50 - 17:30:** Marivone Araujo
- **17:30 - 18:10:** Julia Lamenza - *Iniciando com FinOps: Otimizando Custos na Nuvem*
- **18:10 - 18:20:** Encerramento

## Speakers:
- Patricia Tozi
- Renicius Pagotto Fostaini
- Bianca Veit da Silva
- Julia Pedrosa
- Lucas Ribeiro Mata
- Levi Leopoldino
- Marivone Araujo
- Julia Lamenza

### CODE REVIEW

File: data/events/2024/campinas/main.yml [hunk 1]
```diff
@@ -124,8 +130,14 @@ team_members: # Name is the only required field for team members.
...
-
-
+  - name: "Erik Henrique"
+    role: "Full-Stack Developer"
+    twitter: "theer1k"
+    bio: "Desenvolvedor Full-Stack, formado pela UNISAL com expertise em tecnologias web e desenvolvimento de interface usuário, cursando especialização em GoLang pela FullCycle se aprofundando em arquitetura e soluções na nuvem. Atualmente trabalha como frontend na Canon Medical Systems Brasil."
+    image: "erik.jpg"
+    github: "https://github.com/theer1k"
+    linkedin: "https://www.linkedin.com/in/erikhenriquealvescunha/"
+    website: "https://www.indianboy.com.br/"
```
Comment (toshywoshy) [inline data/events/2024/campinas/main.yml:141]: You're adding an organiser here, this requires @core approval and you to confirm to info

File: static/events/2024-campinas/organizers/erik.jpg
Comment (toshywoshy) [inline static/events/2024-campinas/organizers/erik.jpg:1]: Please move to assets `assets/events/2024-campinas/organizers/`

File: static/events/2024-campinas/speakers/bianca-veit.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/bianca-veit.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/julia-lamenza.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/julia-lamenza.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/renicius-pagotto.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/renicius-pagotto.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/pokemaobr.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/pokemaobr.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/patricia-tozi.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/patricia-tozi.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/marivone-araujo.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/marivone-araujo.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/lucas-ribeiro.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/lucas-ribeiro.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/levi-leopoldino.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/levi-leopoldino.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/events/2024-campinas/speakers/julia-pedrosa.jpeg
Comment (toshywoshy) [inline static/events/2024-campinas/speakers/julia-pedrosa.jpeg:1]: Please move to `assets/events/2024-campinas/speakers/`

File: static/img/sponsors/aws-user-group-campinas.png
Comment (toshywoshy) [inline static/img/sponsors/aws-user-group-campinas.png:1]: Please move to `assets/sponsors/a/`

File: static/img/sponsors/betech.png
Comment (toshywoshy) [inline static/img/sponsors/betech.png:1]: Please move to `assets/sponsors/b/`

File: static/img/sponsors/unisal.png
Comment (toshywoshy) [inline static/img/sponsors/unisal.png:1]: Please move to `assets/sponsors/u/`

**Review by toshywoshy — CHANGES_REQUESTED** [review]

**Comment by toshywoshy** [comment]
Just a general remark, it is better to create smaller PR's we can review in parts.
So one PR for the schedule, another for the sponsors and another for the organiser update, in this way we can move quicker forward with the parts that we can merge in easily.

File: data/events/2024/campinas/main.yml (hunk 1 above)
Comment (theer1k) [reply data/events/2024/campinas/main.yml:141]: https://github.com/devopsdays/devopsdays-web/pull/14683

**Pull request closed without merging**